flint.ui/src/components/Cards/CardInfoRun.vue

```diff
@@ -113,13 +111,17 @@ export default {
       axios
         .get(`${process.env.VUE_APP_REST_API_FLINT_EXAMPLE}/spec`)
         .then((response) => {
-          toast.success(`Specification route has been invoked.`, {
-            timeout: 2000
+          notification['success']({
+            message: 'Specification route has been invoked',
+            duration: 2
```

Comment:
Please also change all the durations to 5, I think 2s is too low and keeping all the notifications to the same duration will provide consistency.